query/src/main/java/org/infinispan/query/logging/Log.java

```diff
@@ -211,4 +212,21 @@ public interface Log extends BasicLogger {
    ParsingException booleanKnnPredicates();
 
    // !!!!!! When adding anything new here please check the last used id in org.infinispan.query.core.impl.Log !!!!!!
+
+   @LogMessage(level = TRACE)
```

Comment:
Trace and debug messages don't go in Log because they should not be i18n-ized

Reply:
I always forget it :( Pushing the fix!